flow-model-generator/modules/flow-model-generator-ls-extension/src/test/resources/search/config/connectors/database.json

```diff
@@ -264,7 +264,7 @@
     },
     {
       "metadata": {
-        "label": "Zoho.crm.rest Client",
+        "label": "Rest Client",
```

Comment:
IMO having only the last part of the package name hierarchy may look ambiguous in many cases. 
As an example, I prefer the name `Zoho CRM Rest Client` in this case. 

Can't we improve our name generation logic accordingly (probably will have to extend the `CONNECTOR_NAME_MAP`)

Reply:
@hasithaa thoughts?

Reply:
Shall we make only the common one for now? Ideally this info should come from the library (or from the library team).

Reply:
> Shall we make only the common one for now? Ideally this info should come from the library (or from the library team).

Yes, getting display names from the packages themselves would be ideal. 
Until then, wouldn’t it be better to add custom mappings for other modules too? (+1 to doing it in a separate PR).

The main reason is that while pro-code users understand package names better, low-code users may find them (especially hierarchical ones) hard to grasp